### PR TITLE
remove space below map on larger screens when the sidebar funnel is active

### PR DESCRIPTION
**Pull request by bmlancien**
see #200 (there is not enough space for all elements to be shown on the sidebar: logos + dropdown)

I'm not sure how to fix this. One idea with this PR is to add a scrollbar to the dropdown elements from the sidebar, so when a funnel is clicked, the content below the funnel doesn't push the logos anymore but they stay in their container. With that, the logos are always visible.

I also slightly increased the width of the sidebar on smaller desktops.

### CODE REVIEW

File: app/templates/index.html [hunk 1]
```diff
@@ -38,205 +38,208 @@ <h2>Nigeria Se4All</h2>
           <img src="{{ url_for('static', filename='logos/logo-2-crest.png') }}">
```
Comment (Bachibouzouk) [inline app/templates/index.html:38]: @bmlancien - did you change anything in this file ? it seems that there is some indent level change but no new code

Comment (bmlancien) [reply app/templates/index.html:38]: I added a div (.sidebar-accordion) to contain all the elements that should be scrollable

**Review by Bachibouzouk — COMMENTED** [review]
Even if I knew that a scroll would be there, it was not intuitive that there were more than 2 filters. 
Is it possible to overflow over the logos? As this would only happen when someone is expanding the filter, I assume they would have seen the logos at first :)

**Comment by bmlancien** [comment]
That would be possible, but that would look weird I think. Like it was a bug from the website.

**Comment by Bachibouzouk** [comment]
> That would be possible, but that would look weird I think. Like it was a bug from the website.

If someone does not see the scroll bar it also looks like a bug from the website that one does not see all the filters, is there a way to highlight it with some magic css trick?

**Comment by bmlancien** [comment]
Maybe it is possible to scroll down to the filter elements after clicking on the funnel.

**Comment by bmlancien** [comment]
Or having a scroll bar for the whole sidebar? We would have to change a few things about the layout but I think that having a scroll bar there would be visible enough for users.

**Comment by christian-rli** [comment]
Hi, I looked at the PR and it took me a while to figure out what was meant. After further inspection I think that neither suggestion is ideal. 

It looks weird to have the normal sidebar upon start show a scrollbar (which it does for me) and you can only move up and down a tiny bit. (buggy look). when selecting a filter one cannot see the filter settings right away and new users would likely not be able to find them. Scrolling down automatically would help with that, but probably also add to the confusion.

In current dev, when the elements are pushed out of the current full page view it also looks buggy, but it's not as confusing.

Is it possible to have the filters expand over the logos on the bottom, but only to cover the full page? In my opinion that would be the best solution (albeit still not ideal). A scrollbar would appear only if the expanded filters would become larger than the height of the screen.

**Comment by bmlancien** [comment]
I made the sidebar scrollable (for desktop screens) when the content is bigger than the window height. It should push the logos down and not overlap them @christian-rli Have a look at it. I think this is better to push them down than to overlap them. Pushing them down doesn't look like a bug.

**Comment by christian-rli** [comment]
Thank you @bmlancien ! You're right pushing down looks better. I'm happy with the way it looks now. Just one question: is is possible to hide the scroll bar if there is nothing to scroll? If no, I'm also fine with merging this @Bachibouzouk

**Comment by bmlancien** [comment]
It is possible. But when the scroll bar appears, the width for the content in the sidebar gets smaller (because the scroll bar is like 15px wide) and so it get squeezed a bit. The offset is not super nice, that's why I thought of always keeping the scrollbar present. But let me check if there is a way to keep the content width consistent with and without scroll bar.

**Comment by bmlancien** [comment]
@christian-rli Now the scroll-bar is visible only when needed. I'm not sure what is best. I guess it depends on how often the users will want to see the filters. If often, the shift in the content due to with/without scroll-bar might be annoying. If not often, having the ugly scroll-bar is not necessary...

**Comment by christian-rli** [comment]
> The offset is not super nice,

True, but I think I prefer it that way. We can change it again, if we get word from above. Thanks again.

**Comment by christian-rli** [comment]
I guess we wrote on the same instance @bmlancien :) Anyway, let's keep what we have now.

**Review by Bachibouzouk — APPROVED** [review]
I find it more intuitive to have the scroll bar from the beginning if it is needed, so I like the proposed last solution :)

**Pull request merged**